pom.xml

```diff
@@ -79,9 +79,11 @@
         <dependency>
             <groupId>javax</groupId>
             <artifactId>javaee-api</artifactId>
-            <version>7.0</version>
+            <version>8.0</version>
```

Comment:
IMO, the original proposal was better.

Reply:
Sorry for the confusion. I meant the original idea of specific dependencies.

I just did the "negative" review so this is not merged until we reach consensus.